tests/utils/index.js

```diff
@@ -1111,6 +1111,7 @@ const waitForDockerLogs = (container, ...regex) => {
 };
 
 const waitForApiLogs = (...regex) => waitForDockerLogs('api', ...regex);
+const waitForSentinelLogs = (...regex) => waitForDockerLogs('sentinel', ...regex);
```

Comment:
Minor: should `waitForSettingsUpdateLogs` be updated to use this?

Reply:
Done